pkgs/development/tools/build-managers/gradle/default.nix

```
@@ -23,104 +23,133 @@ rec {
         "x86_64-darwin"
         "x86_64-linux"
         "x86_64-windows"
-      ]
+      ],
+
+      # Set to true if this Gradle version supports JVM toolchains
+      # (see here for more info: https://docs.gradle.org/current/userguide/toolchains.html#sec:consuming).
+      #
+      # Toolchains were added in Gradle 6.7. For >=6.7 versions, set this to
+      # true (or false if you want to disable this package's toolchain support).
+      # For <6.7 versions, set this to false or passthru.tests will fail!
+      #
+      # If true, this will enable the 'toolchains' attreibute and generate a
+      # test in passthru.tests that verifies that:
+      # a) The package can be used with Java toolchains via the javaToolchains
+      #    argument
+      # b) Gradle correctly picks up toolchains managed via this package
+      hasToolchainSupport ? false
```

```suggestion
      hasToolchainSupport ? lib.versionOlder version "6.7"
```

(Haven't checked the condition is the right operator) and you need to add lib somewhere